debian/packages/get-kernel.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+#
+# get-kernel - Downloads Ubuntu kernel packages
+#
+# Author: Alexsander de Souza <alexsander.souza@canonical.com>
+#
+# Copyright (C) 2022 Canonical
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+export LANG=C
+
+KERNEL=${KERNEL:-linux-image-generic}
+ARCH=amd64
```

Comment:
call `dpkg --print-architecture` instead

Reply:
Removed as not being used.